sdks/models/src/main/avro/core/grscicoll-record.avsc

```diff
@@ -0,0 +1,119 @@
+[
+  {
+    "name" : "MatchType",
+    "namespace" : "org.gbif.pipelines.io.avro.grscicoll",
+    "type" : "enum",
+    "symbols" : ["EXACT", "FUZZY", "MACHINE_TAG", "NONE"]
+  },
+  {
+    "name" : "Status",
+    "namespace" : "org.gbif.pipelines.io.avro.grscicoll",
+    "type" : "enum",
+    "symbols" : [
+      "ACCEPTED", "AMBIGUOUS", "AMBIGUOUS_MACHINE_TAGS", "AMBIGUOUS_OWNER", "AMBIGUOUS_INSTITUTION_MISMATCH", "DOUBTFUL"
+     ]
+  },
+  {
+    "name" : "Reason",
+    "namespace" : "org.gbif.pipelines.io.avro.grscicoll",
+    "type" : "enum",
+    "symbols" : [
+       "CODE_MATCH", "IDENTIFIER_MATCH", "ALTERNATIVE_CODE_MATCH", "NAME_MATCH", "PROBABLY_ON_LOAN", "INST_COLL_MISMATCH",
+       "COUNTRY_MATCH", "INSTITUTION_TAG", "COLLECTION_TAG", "COLLECTION_TO_INSTITUTION_TAG", "INSTITUTION_TO_COLLECTION_TAG"
+    ]
+  },
+  {
+    "name" : "IdentifierType",
+    "namespace" : "org.gbif.pipelines.io.avro.grscicoll",
```

Comment:
I think we should try to avoid ENUMs in avro, we will have 2 places in total for ENUMs -> gbif.api and avro, I see 2 possible issues:
1) it is highly possible to get an error when we add/change an ENUM in gbif-api library and forgot about ENUMs in avro, and error we can find in runtime only
2) What if we have avro files with old version of ENUMs and we edited ENUMs

All in all, I think we should store only ENUM's name and parse them in code